lib/init/init-content.js

```diff
@@ -20,30 +21,44 @@ function splitAndRejoin(fn) {
   );
 }
 
+function getContentByLang(lang) {
+  if (content[lang]) {
+    return content[lang];
+  }
+  return content.EN;
+}
+function setContentLanguage(lang, cb) {
```

Comment:
Instead of setting the language at one point in the code by calling `setContentLanguage` and then the rest of the code depending on that call, I much prefer the lang being passed around in the code (either directly or in a config object as I think was done in much of the rest of the code). That makes the code much more predictable and less surprising.

I would remove this function, and add a config parameter that contains the lang wherever needed.